texas-hearing-institute/pages/Onboarding/OnboardingPage.tsx

```diff
@@ -83,7 +96,8 @@ const Onboarding: React.FC<{ route: any; navigation: any }> = ({
 				) : (
 					<Pressable
 						onPress={() =>
-							navigation.navigate(`Onboarding${pageNumber}` as any, {
+							// FIX THIS @baylee
+							navigation.navigate(`Onboarding1`, {
```

Comment:
react navigation type checking doesn't work with this dynamic page naming scheme, let's just create separate pages for these... only two of the pages share components anyway

Reply:
sure that sounds good